app/games/simulation-engine.ts

```
@@ -0,0 +1,516 @@
+            minute,
+            type: 'goal',
+            description: `⚽ Opposition scores!`,
+            homeTeam: false,
+            importance: 'high'
+          });
+        }
+        return this.events[this.events.length - 1];
+        
+      case 'miss':
+        const missPlayer = this.selectRandomPlayer('attacker');
+        this.events.push({
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)